specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10292 @@
+          "description": "The reference name of the integration runtime."
+        },
+        "integrationRuntimeType": {
+          "type": "string",
+          "description": "The type of the integration runtime.",
+          "readOnly": true
+        }
+      }
+    },
+    "ScanProperties": {
+      "type": "object",
+      "description": "scan properties",
+      "properties": {
+        "scanRulesetName": {
+          "type": "string",
+          "description": "The name of the scan ruleset."
+        },
+        "businessRuleSetName": {
+          "type": "string",
+          "description": "The name of the business rule set."
+        },
+        "scanRulesetType": {
+          "enum": [
+            "Custom",
+            "System"
+          ],
+          "type": "string",
+          "description": "The type of the scan ruleset.",
+          "x-ms-enum": {
+            "name": "ScanRulesetType",
+            "modelAsString": true
+          }
+        },
+        "collection": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/CollectionReference"
+            }
+          ],
+          "description": "The collection reference of the scan."
+        },
+        "domain": {
+          "type": "string",
+          "description": "The domain of the scan."
+        },
+        "workers": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of workers of the scan."
+        },
+        "createdAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The creation time of the scan.",
+          "readOnly": true
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "The last modified time of the scan.",
+          "readOnly": true
+        },
+        "connectedVia": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ConnectedVia"
+            }
+          ],
+          "description": "The integration runtime reference of the scan."
+        },
+        "isPresetScan": {
+          "type": "boolean",
+          "description": "The flag to indicate whether the scan is a preset scan or not."
+        },
+        "isLiveViewEnabled": {
+          "type": "boolean",
+          "description": "The flag to indicate whether the live view is enabled or not."
+        },
+        "parallelScanCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of parallel scans."
+        },
+        "logLevel": {
+          "type": "string",
+          "description": "The log level of the scan."
+        }
+      }
+    },
+    "ExpandingResourceScanProperties": {
+      "type": "object",
+      "description": "expanding resource scan properties",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ScanProperties"
+        }
+      ],
+      "properties": {
+        "resourceTypes": {
```

Comment:
Make it a named schema in definitions, e.g. `#/definitions/ResourceTypes` (you name it).

Reply:
Sure, I changed it to: #/definitions/ExpandingResourceScanPropertiesResourceTypes